site/en/guides/build/index.md

```diff
@@ -48,32 +15,27 @@ The most common workflow is:
             $ docker pull openthread/environment:latest
             $ docker run -it --rm openthread/environment bash
 
-1.  Within your chosen environment, clone the OpenThread Git repository:
+1.  Within your chosen environment, clone the platform-specific OpenThread Git repository. Taking nrf52840 as an example:
```

Comment:
Maybe we should stick to cc2538 for now until  we can support the different platforms in the docs.